README.md

```diff
@@ -19,7 +19,7 @@ Here you'll find all the core components you need to create a delightful webapp.
 1 - Install the **peer dependencies** listed below:
 
 ```bash
-yarn add @xstyled/styled-components react styled-components
+yarn add @wttj/xstyled-styled-components react styled-components
```

Comment:
is it public ?

Reply:
Not yet but it's will be discuss tomorrow :)